### ISSUE TITLE
Dialog not working with Vue 3

### ISSUE THREAD

**Issue by ifarooq** [Godofbrowser/vuejs-dialog#78]
Dialog not working with Vue 3

**Comment by TheJaredWilcurt** [comment]
@Godofbrowser any plans for a Vue 3 compatible version of vuejs-dialog?

**Comment by Godofbrowser** [comment]
@TheJaredWilcurt @ifarooq Thanks for continued interest in VuejsDialog. Although I had to dedicate my time to other things at some point, I have the intention of not only making this work with Vue3 but also a rewrite of some parts of the plugin to address issues from the previous versions. As a first step, I'd bootstrap the plugin this weekend for Vue3 and have an estimate of how much effort it will take to complete. Will update this issue.

**Comment by ifarooq** [comment]
Ok thanks much.

**Comment by Korko** [comment]
Sorry to bother but, any news?

**Comment by Godofbrowser** [comment]
Yes, I resumed work on vue3 migration. Sorry for the long wait https://github.com/Godofbrowser/vuejs-dialog/tree/dev-v3
